src/xlsx/extraction/chart_extractor.ts

```diff
@@ -139,13 +144,33 @@ export class XlsxChartExtractor extends XlsxBaseExtractor {
                 required: true,
               })!,
               backgroundColor: color ? `${toHex(color.asString())}` : undefined,
+              trend: this.extractChartTrendline(chartDataElement),
             };
           }
         );
       })
       .flat();
   }
 
+  private extractChartTrendline(
+    chartDataElement: Element
+  ): ExcelChartTrendConfiguration | undefined {
+    const trendlineElement = this.querySelector(chartDataElement, "c:trendline");
+    if (!trendlineElement) {
+      return undefined;
+    }
+    const trendlineType = this.extractChildAttr(trendlineElement, "c:trendlineType", "val");
+    const trendlineColor = this.extractChildAttr(trendlineElement, "a:solidFill a:srgbClr", "val");
+    const trendlineOrder = this.extractChildAttr(trendlineElement, "c:order", "val");
```

Comment:
Do we always need to extract the order ? Maybe we can do it only if we are in the "polynomial" case ?